public/app/features/serviceaccounts/ServiceAccountPage.tsx

```diff
@@ -42,17 +43,8 @@ function mapStateToProps(state: StoreState) {
     timezone: getTimeZone(state.user),
   };
 }
-const mapDispatchToProps = {
-  loadServiceAccount,
-  loadServiceAccountTokens,
-  createServiceAccountToken,
-  deleteServiceAccountToken,
-  deleteServiceAccount,
-  updateServiceAccount,
-  fetchACOptions,
-};
 
-const connector = connect(mapStateToProps, mapDispatchToProps);
+const connector = connect(mapStateToProps);
```

Comment:
Feels a bit weird that we're using half of the old way to connecting components to redux and half the new. Any particular reason to switch to redux hooks here? It'll make testing this component harder.